### DISCUSSION TITLE
Official Helm chart based on kube-thanos

### DISCUSSION THREAD

**Discussion by stevehipwell** [thanos-io/thanos#4475] [Ideas & Features]
I'm interested in an official Helm chart based on the code in [kube-thanos](https://github.com/thanos-io/kube-thanos) and am happy to contribute to get this created. I have the following potential solutions, in no particular order, listed below.

- New `thanos-io/helm-charts` repo with `kube-thanos` chart added (based off kube-thanos source)
- Add charts folder to kube-thanos and create the chart there
- Create the chart at https://github.com/prometheus-community/helm-charts/, as kube-state-metrics have done

If none of these options are desirable/possible I will be creating a chart in my own [repo](https://github.com/stevehipwell/helm-charts) but this is only a fall-back option.

**Comment by zhekazuev** [comment]
@stevehipwell Hi,

Can I use your Thanos Helm Chart as a base and try to add it to one of the Thanos official repos(thanos-io/thanos or thanos-io/kube-thanos)?

Bitnami not an option for me, but private repository too. It should be community-based one.

Best regards!

**Reply by stevehipwell** [reply]
@zhekazuev you're more than welcome to take my chart as a base, it's intentionally MIT licenced. I would caution that you only want to do this if you have maintainers and the governance structure in place to support it.

**Reply by zhekazuev** [reply]
@stevehipwell Thanks!

**Comment by zhekazuev** [comment]
A list of related Issues and links to mention.

Issues:
- https://github.com/thanos-io/thanos/issues/8381
- https://github.com/thanos-io/thanos/issues/7204
- https://github.com/thanos-io/thanos/issues/1820
- https://github.com/thanos-io/kube-thanos/issues/316
- https://github.com/prometheus-community/helm-charts/issues/4706
- https://github.com/prometheus-community/helm-charts/issues/1169
- https://github.com/bitnami/charts/issues/36215#issuecomment-3233843307

Charts:
- https://github.com/thanos-io/kube-thanos/tree/main/manifests
- https://github.com/bitnami/charts/tree/main/bitnami/thanos
- https://github.com/stevehipwell/helm-charts/tree/main/charts/thanos
- https://github.com/banzaicloud/banzai-charts/tree/master/thanos
- https://github.com/solidcharts/helm-charts/tree/main/charts/thanos
- https://github.com/thanos-community/helm-charts

Common:
- https://github.com/thanos-io/thanos/tree/main/tutorials/kubernetes-helm
- https://github.com/thanos-community/thanos-operator
- https://github.com/prometheus-operator/prometheus-operator/discussions/6745
- https://github.com/thanos-io/thanos/discussions/8275
